src/App.jsx

```diff
@@ -62,15 +62,11 @@ export function App() {
 		// eslint-disable-next-line react-hooks/exhaustive-deps
 	}, [listToken]);
 
-	function logOut() {
-		const confirm = window.confirm(
-			`If you're ready to log out, make sure to write down your list name before you click ok! It is "${listToken}".`,
-		);
-		if (confirm) {
-			localStorage.clear();
-			setListToken('');
-			navigateTo('/');
-		}
+	if (confirmLogOut === true) {
```

Comment:
Refactor (nitpick): No need to compare to true.

```suggestion
	if (confirmLogOut) {
```